### DISCUSSION TITLE
Shared memory transport support?

### DISCUSSION THREAD

**Discussion by He-Pin** [netty/netty#14624] [General]
As Netty 4.2 splits the io handler and the event loop, I think a shared memory transport may be possible; use case: SSR.

Background:
We have a Worker which is implemented with CPP, and makes use of V8 to do Server Side Rendering(SSR),
And a Java sidecar as Data integrating panel, which communities with other internal services and the Worker (Sending requests to the Worker and the Worker response Rendered Mobile WebPages)
The Java Data integrating panel uses Netty to communicate with the Worker(Written in CPP) by Unix domain sockets.
It works well, but sharing memory should be the best performance.

**Comment by normanmaurer** [comment]
Not sure I get it ... can you give more details ?

**Reply by He-Pin** [reply]
I have updated the details, but still not exploring it.
